t/op/repeat.t

```diff
@@ -193,6 +193,14 @@ fresh_perl_like(
 eval q{() = (() or ((0) x 0)); 1};
 is($@, "", "RT #130247");
 
+# [GH #13324] Perl croaks if a string repetition seems unsupportable
+fresh_perl_like(
+ 'use warnings; my $x = "A" x (2**99)',
```

Comment:
can we try some maximum toxic 0.0 NV literals here? maybe creating them with PP pack and or unpack.